core/controller/src/main/scala/whisk/core/controller/WebActions.scala

```diff
@@ -743,6 +758,16 @@ trait WhiskWebActionsApi extends Directives with ValidateRequestSize with PostAc
     }
   }
 
+  /**
+   * Checks if an action is executable.
+   */
+  private def checkEntitlement(identity: Identity, action: WhiskActionMetaData)(
+    implicit transid: TransactionId): Future[Unit] = {
+    val resource =
+      Resource(action.namespace.root.toPath, Collection(Collection.PACKAGES), Some(action.namespace.last.toString))
+    entitlementProvider.check(identity, Privilege.READ, resource)
```

Comment:
this check also checks the throttles - which means there's now a redundant check for throttles.
the code (before this pr), did not check entitlement through the provider because it can implicitly infer the owner (there's no indirection through a binding) so it only checked throttles.

Reply:
Thank you for your kind review. I've removed existing entitlementProvider.checkThrottles(), then modified to use only entitlementProvider.check ().